.github/workflows/release.yaml

```diff
@@ -0,0 +1,56 @@
+name: create release on new tags
+
+on:
+  push:
+    # Publish semver tags as releases.
+    tags: [ '*.*.*' ]
+      # [0-9]+.[0-9]+.[0-9]+
+
+jobs:
+  build-and-push:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Build binary and make tarball
+        id: build_bin
+        run: |
+          make bin
+          tar -zcvf chart-verifier.tgz -C out/ chart-verifier
+          export TARBALL_PATH=$(realpath chart-verifier.tgz)
+          echo "tarball_path=$TARBALL_PATH"
+
+      - name: Print tag to GITHUB_OUTPUT
+        id: get_tag
+        run: |
+          echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_OUTPUT
```

Comment:
I don't remember exactly what shows up in GitHub Action logs, but if this won't get displayed to stdout, maybe we should use `tee -a` instead of `>>` so that we also see it in stdout of this task.

Reply:
That's a good idea, I changed it to use "tee -a"